### PR TITLE
docs: add security advisory

### PR DESCRIPTION
**Pull request by raymondfeng**
Signed-off-by: Raymond Feng <enjoyjava@gmail.com>

### CODE REVIEW

File: pages/en/sec/Security-advisory-11-20-2020.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+> Prototype Pollution is a vulnerability affecting JavaScript. Prototype Pollution refers to the ability to inject properties into existing JavaScript language construct prototypes, such as objects. JavaScript allows all Object attributes to be altered, including their magical attributes such as `_proto_`, `constructor` and `prototype`. An attacker manipulates these attributes to overwrite, or pollute, a JavaScript application object prototype of the base object by injecting other values. Properties on the Object.prototype are then inherited by all the JavaScript objects through the prototype chain. When that happens, this leads to either denial of service by triggering JavaScript exceptions, or it tampers with the application source code to force the code path that the attacker injects, thereby leading to remote code execution.
+>
+> There are two main ways in which the pollution of prototypes occurs:
+>
+> - Unsafe Object recursive merge
+> - Property definition by path
+
+### The Fix
+
+We improve JSON parsing to reject payload that contains `malicious` attributes.
```
Comment (emonddr) [inline pages/en/sec/Security-advisory-11-20-2020.md]:
```suggestion
We improved JSON parsing to reject a payload that contains `malicious` attributes.
```

File: pages/en/sec/Security-advisory-11-20-2020.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+```json
+"dependencies": {
+   ...
+   "@loopback/rest": "^9.0.0",
+   ...
+ },
+```
+
+You can install latest version of `@loopback/cli` and run `lb4 update` command:
```
Comment (emonddr) [inline pages/en/sec/Security-advisory-11-20-2020.md]:
```suggestion
Install the latest version of `@loopback/cli` and run the `lb4 update` command:
```

**Review by emonddr — CHANGES_REQUESTED** [review]

**Review by dhmlau — APPROVED** [review]
Thanks @jannyHou. Please rebase and squash the commits before merging.

**Review by emonddr — APPROVED** [review]

**Comment by attritionorg** [comment]
In the advisory it includes:
 **Security risk**: High (CVSS: 7.3)

Could someone include the full CVSS string that shows the derivation of this score please?

**Pull request merged**